ee-9/feature-pack/src/main/resources/feature_groups/standalone-full.xml

```diff
@@ -3,6 +3,7 @@
 
     <feature-group name="standalone-full-x-messaging"/>
     <feature-group name="messaging-remote-activemq"/>
+    <feature-group name="microprofile-opentracing-jaeger"/>
```

Comment:
@bstansberry is that intended?

I thought that MP specs would be made available only in the `standalone-microprofile` configurations even for EE9

Reply:
@jmesnil @ehsavoie Good question. Let's discuss when we talk in a few minutes and we can post back here.

Emmanuel may have had this all along or may have added it in response to my request that he compare outputs and eliminate diffs.  But that was based on my unthinking idea that these specs are in the 'wildfly' and 'wildfly-preview' FPs so there's no reason the resulting configs should change. But they do, correctly I think, when it comes to health and metrics, so why not differ about this too?

Reply:
@jmesnil @bstansberry yes the goal was to have a minimal diff with the preview release. Maybe removing this from preview should come in a 2nd step ?

Reply:
ok, let's make the diff minimal and define the rules for inclusion before this preview distribution is finalized

Reply:
@jmesnil @ehsavoie +1 to make the diff minimal for this OT change.

A summary, hopefully correct, of what we discussed.

1) The 'wildfly' and 'wildfly-preview' FPs were not meant to differ re: MP. Both contain the MP extensions and unlike, say, legacy security, we're not planning to drop the extensions. We could use wildfly-preview to preview some change in how we offer MP in our standard configs but that wasn't the original intent.

2) It's a valid question whether we want to continue offering the MP specs in our main distribution's standard configs other than standalone-microprofile[-ha].xml. That discussion would also encompass MP JWT, which we include in standalone[-xxx] and the various domain.xml profiles.

But changing those things seems beyond the scope of what this PR is about.